pkg/kubelet/dockertools/docker_test.go

```diff
@@ -698,10 +698,15 @@ func TestMakePortsAndBindings(t *testing.T) {
 			HostPort:      445,
 			Protocol:      "foobar",
 		},
+		{
+			ContainerPort: 443,
+			HostPort:      446,
+			Protocol:      "tcp",
+		},
 	}
 	exposedPorts, bindings := makePortsAndBindings(ports)
 	if len(ports) != len(exposedPorts) ||
-		len(ports) != len(bindings) {
+		len(ports) == len(bindings) {
```

Comment:
This checks becomes meaningless here....can you check whether there are redundant bindings?